BoxContentSDK/BoxContentSDK/Operations/BOXStreamOperation.m

```diff
@@ -148,7 +148,9 @@ - (long long)contentLength
 
 - (void)abortWithError:(NSError *)error
 {
+    self.error = error;
     [self.sessionTask cancel];
+    //FIXME: Remove?
```

Comment:
I actually meant to call this out in the PR. I'd like to resolve this before pushing ideally.

I think this was from a previous change I made, where I called the finish directly because the URLConnection code was doing it like that before, however as of a recent change I made for the same thing in another class, I don't think we need to. I'll run another test and also wanted your guys' thoughts in case I was missing something.